src/handlers/shared/start.ts

```diff
@@ -44,7 +46,7 @@ export async function start(context: Context, issue: Context["payload"]["issue"]
 
   // check for max and enforce max
 
-  if (assignedIssues.length - openedPullRequests.length >= maxConcurrentTasks) {
+  if (assignedIssues.length - openedPullRequests.length >= maxConcurrentTasks && !isUserMember(context, assignee)) {
```

Comment:
According to the spec, this should be configurable.

Reply:
about the configurability, what are the various user roles possible and what should be their maxConcurrentTasks limits. Please